config/defaults.conf

```diff
@@ -228,6 +228,16 @@ bert_fine_tune = 0  // If false, BERT transformer weights will
                     // high amount of GPU memory, so consider
                     // lowering the batch size if set to true.
 
+
+onlstm_chunk_size = 10  // Chunk downsizing factor for ON-LSTM master gate, dimensions
+                        // of master gate: D/C where C is the chunk downsizing factor.
+onlstm_dropconnect = 0.5  // Linear dropout between ON-LSTM layers.
+onlstm_dropouti  = 0.3  // Locked Dropout on input embeddings.
+onlstm_dropouth  = 0.3  // Locked Dropout between ON-LSTM layers.
+onlstm_tying = 0  // Language Modeling tying of weights.
+
+wsj_parsing = 0  // Set if using for PTB parsing task. Makes sure to include WSJ vocabulary.
```

Comment:
There could be a situation where it may not be a PTB parsing task but may use the same task structure, for example, doing dev F1 on MNLI. In that case, we wouldn't want to include the WSJ vocabulary by force.

Reply:
Agreed. Changed name, should be easier to comprehend now.